test/api_client/api_resource.js

```diff
@@ -37,8 +36,8 @@ describe('API resource module', () => {
 	};
 
 	const sendRequestResult = {
-		status: 200,
-		body: { success: true, sendRequest: true },
+		data: [],
```

Comment:
`data.data`? Why not define the whole result here instead of partly here and partly in the stub `resolves` call?

Reply:
first `data` is the response by `axios`, so thats why I wanted to separate them.
`sendRequestResult` represents the actual response from the servere